backend/package.json

```diff
@@ -30,6 +30,15 @@
     "scratch:download:types:schema": "curl -o .temp/scratch/sb3_schema.json https://raw.githubusercontent.com/scratchfoundation/scratch-parser/refs/heads/master/lib/sb3_schema.json",
     "scratch:download:types": "yarn mkdirp .temp/scratch && yarn scratch:download:types:definitions && yarn scratch:download:types:schema",
     "scratch:update-types": "yarn scratch:download:types && cd .temp/scratch && json2ts -i sb3_schema.json -o ../../src/ast/types/input/scratch/generated/sb3.ts",
+    "jupyter:download:types:schema": "curl -o .temp/jupyter/nbformat.json https://raw.githubusercontent.com/jupyter/nbformat/refs/heads/main/nbformat/v4/nbformat.v4.schema.json",
+    "jupyter:download:types": "yarn mkdirp .temp/jupyter && yarn jupyter:download:types:schema",
+    "jupyter:update-types": "yarn jupyter:download:types && cd .temp/jupyter && json2ts -i nbformat.json -o ../../src/ast/types/input/jupyter/generated/nbformat.ts",
+    "antlr:download:grammers": "yarn mkdirp .temp/antlr && cd .temp/antlr && git clone https://github.com/antlr/grammars-v4.git",
```

Comment:
To get proper version control, should we make the antlr grammars-v4 a submodule? This way we can choose when to update it or not. Otherwise builds may not be reproducible.